enterprise/frontend/src/metabase-enterprise/sandboxes/components/AttributeMappingEditor/AttributeMappingEditor.tsx

```diff
@@ -56,8 +56,7 @@ const AttributeMappingEditor = ({
         {shouldUseSavedQuestion ? t`Parameter or variable` : t`Column`}
       </div>
     }
-    // eslint-disable-next-line @typescript-eslint/ban-ts-comment
-    // @ts-ignore
+    // @ts-expect-error type is incorrect
```

Comment:
This case can be fixed by returning an empty fragment (`<Fragment />`) instead of `null`

Reply:
fixed